src/git.rs

```diff
@@ -34,13 +34,20 @@ fn lookup_rev<'rev>(repo: &'rev Repository, rev: &str) -> Result<Git2Commit<'rev
 
 fn get_repo() -> Result<Repository, Error> {
     fn open(repo: &Path) -> Result<Repository, Error> {
+        eprintln!("refreshing repository at {:?}", repo);
+        // This uses the CLI because libgit2 is quite slow to fetch a large repository.
+        let status = std::process::Command::new("git")
+            .arg("fetch")
```

Comment:
I think this needs to use something like the old `repo.remote_anonymous(RUST_SRC_URL)`. E.g., if I run `git fetch` in my rust-lang/rust clone, it only fetches my fork, which doesn't have the latest changes.

Reply:
Good point!  I normally point it at a repo where origin is rust-lang/rust, but it makes sense to use the correct remote.  I don't think it can use an anonymous remote (since that is in memory only), but I added the URL to the fetch line which should do the right thing.